website/docs/registries/overview.md

```diff
@@ -0,0 +1,25 @@
+# Overview
+
+Scarb is not only a Cairo build toolchain but also a package manager.
+It uses registries to store and manage packages.
+While Scarb supports alternative ways to host packages, registries make it easy to publish, discover, and integrate packages into your projects.
+
+## Official registry
+
+Currently, the default way to host packages is via the official [scarbs.xyz](https://scarbs.xyz) registry.
+Please note that the official registry is still in development.
+You can already use it to discover and [add](#adding-dependencies) packages to your projects.
+[Publishing](#packaging-and-publishing) packages is currently limited, but if there are any other packages you would like to be available - please reach out to us on [Telegram](https://t.me/scarbs_xyz) or [Discord](https://discord.gg/7YXj4Z2).
+
+## Custom registry
```

Comment:
This should be last here, it seems really exotic to me

Reply:
Yeah, I suppose that makes sense, addressed in a372664c0a24dffc572a231b10077b8de85eed0f